docs/docs/docs/01-core/content-releases/01-backend.md

```diff
@@ -197,6 +197,34 @@ Exposes methods to schedule release date for releases.
 packages/core/content-releases/server/src/services/scheduling.ts
 ```
 
+### Release status update triggers:
+
+Considering that retrieving the status of all entries in a release is a heavy operation, we don't fetch it every time a user wants to access a release. Instead, we store the status in a field within the Release Content Type, and we only update it when an action that changes the status is triggered. These actions include:
```

Comment:
```suggestion
In order to minimize the overhead of retrieving the status of all entries within a release, we have implemented an optimization strategy. Instead of fetching the status every time a user accesses a release, we store it in a dedicated field within the Release Content Type. This status field is updated only when specific actions occur that necessitate a change in status. These actions include:
```